mlibrary/lib.umich.edu
# May 2023 dependency updates.

# Overview
Monthly dependency update to help maintain lib.umich.edu.

## NPM
These dependencies have been updated to their latest versions:
- `@emotion/react`
- `@emotion/styled`
- `entities`
- `fetch-retry`
- `gatsby`
- `gatsby-plugin-emotion`
- `gatsby-plugin-image`
- `gatsby-plugin-manifest`
- `gatsby-plugin-sharp`
- `gatsby-plugin-sitemap`
- `gatsby-source-drupal`
- `gatsby-source-filesystem`
- `gatsby-transformer-sharp`
- `patch-package`
- `prettier`
- `rehype-react`
- `use-debounce`

## Testing
- Install the updated packages (`npm install`).
- Start [the site](http://localhost:8000) and check if everything still works (`npm start`).
- Make sure the PR is consistent in these browsers:
